packages/asl-projects/assets/scss/projects.scss

```diff
@@ -1107,14 +1107,22 @@ html.modal-open, html.modal-open body {
     &:last-child {
       border-bottom: 1px solid $govuk-border-colour;
     }
-
     header {
       padding: $govuk-gutter-half;
       padding-left: 0;
       position: relative;
       cursor: pointer;
       margin: 0;
 
+      .govuk-button.link {
+        width: 100%;
+        text-align: left;
+
+        h3:after {
+          top: 0
+        }
+      }
+
```

Comment:
This needs to specifically target those in the details accordion. When the button in the protocol header is full width, the floating status badge pushes it down if it's also full width.